sugarloaf/src/font/mod.rs

```diff
@@ -536,14 +536,7 @@ impl SharedData {
     /// Creates shared data from the specified bytes.
     pub fn new(data: Vec<u8>) -> Self {
         Self {
-            inner: Arc::from(data),
-        }
-    }
-
-    /// Creates shared data from an Arc<Vec<u8>> (avoids cloning)
-    pub fn from_arc(data: Arc<Vec<u8>>) -> Self {
-        Self {
-            inner: Arc::from(data.as_ref().clone()),
+            inner: Arc::from(data.into_boxed_slice()),
```

Comment:
```suggestion
            inner: Arc::from(data),
```

std has [an impl](https://doc.rust-lang.org/std/sync/struct.Arc.html#impl-From%3CVec%3CT,+A%3E%3E-for-Arc%3C%5BT%5D,+A%3E) so you don't have to make an intermediate boxed slice (though I doubt this'll actually affect codegen at all, it's just easier to read imo)